src/Stratis.Bitcoin.LightD/Program.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Threading.Tasks;
+using Stratis.Bitcoin.Builder;
+using Stratis.Bitcoin.Configuration;
+using Stratis.Bitcoin.Features.BlockStore;
+using Stratis.Bitcoin.Features.Consensus;
+using Stratis.Bitcoin.Features.MemoryPool;
+using Stratis.Bitcoin.Features.RPC;
+using Stratis.Bitcoin.Features.Wallet;
+using Stratis.Bitcoin.Features.Api;
+using Stratis.Bitcoin.Features.Apps;
+using Stratis.Bitcoin.Features.Miner;
+using Stratis.Bitcoin.Utilities;
+using Network = Stratis.Bitcoin.Networks.Networks;
+
+namespace Stratis.Bitcoin.LightD
+{
+    public class Program
+    {
+        public static async Task Main(string[] args)
+        {
+            try
+            {
+                var nodeSettings = new NodeSettings(networksSelector: Network.Bitcoin, args: args);
+
+                IFullNode node = new FullNodeBuilder()
+                    .UseNodeSettings(nodeSettings)
+                    .UseBlockStore()
```

Comment:
Should this be the pruned store? Similarly to what is used by the light wallet feature.

Reply:
Updated to use `LightWallet` which uses PrunedStore

Reply:
`UseBlockStore`
`AddMining`
`UseApps`
All this can go